### PR TITLE
Add teamTemplate cmdlets to the available list

### PR DESCRIPTION
**Pull request by levdavid**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@brandber PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by levdavid** [comment]
Is there anything that we need addressed before merging?

**Comment by scanum** [comment]
@rajgodwin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by levdavid** [comment]
@rajgodwin pinging since this still was not merged

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -43,6 +43,62 @@ This page provides the latest Teams PowerShell change log for both General Avail
 | March 2020 | [1.0.5](https://www.powershellgallery.com/packages/MicrosoftTeams/1.0.5) |<li>Added New-CsBatchPolicyAssignmentOperation</li> |
 | Feb 2020 | [1.0.4](https://www.powershellgallery.com/packages/MicrosoftTeams/1.0.4) | <li>Get-Team optimizations</li>  |
 
+### Cmdlet availability
+
+> [!NOTE]
+> The list in the table below only includes cmdlets that are natively part of the Teams PowerShell module. The Teams cmdlets in the S[kype for Business Online Connector module](/powershell/skype/intro?view=skype-ps) are not displayed. However, as those cmdlets are migrated natively into Teams PowerShell, we'll add them to this table.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:49]:
```suggestion
> The list in the following table only includes cmdlets that are natively part of the Teams PowerShell module. The Teams cmdlets in the [Skype for Business Online Connector module](/powershell/skype/intro?view=skype-ps) are not displayed. However, as those cmdlets are migrated natively into Teams PowerShell, we'll add them to this table.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@levdavid, could you kindly commit the suggested changes before merging? Many thanks.

**Comment by levdavid** [comment]
@JohanFreelancer9  done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
This article has been updated since this PR was submitted. Closing this PR.

cc: @scanum

**Pull request closed without merging**